superwoman7/js/app.js

```diff
@@ -14,11 +40,5 @@ app.controller('tweet-search', function($scope, $http, $sce) {
 			alert('Oh no! An error! '+error);
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/superwoman7/js/app.js
+++ b/superwoman7/js/app.js
@@ -37,7 +37,7 @@
 			$scope.searchTerm = $scope.search
 			$scope.loading = false;
 		}, function(error) { 
-			alert('Oh no! An error! '+error);
+            alert('Oh no! An error! '+error);
 		});
 	}
 });
```